tests/test_vector2_truncate.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from ppb_vector import Vector2
+
```

Comment:
Not enough white space, please see [PEP8](https://www.python.org/dev/peps/pep-0008/).